src/TeamSpeak3QueryApi/Responses/ClientBan.cs

```diff
@@ -1,4 +1,4 @@
-namespace TeamSpeak3QueryApi.Net.Specialized.Responses
+namespace TeamSpeak3QueryApi.Net.Responses
```

Comment:
Parsed responses of a rich client do not belong to the same namespace as the query client.